### DISCUSSION TITLE
“Firefox”?

### DISCUSSION THREAD

**Discussion by Makitoid** [zen-browser/desktop#1964] [Ideas]
![1.png](https://github.com/user-attachments/assets/2a88a992-e6c3-4d07-b19e-26a1b21c7767)
I think it's not proper to display _Firefox _ in such places
I just feel uncomfortable with seeing this...as if I was using firefox
is it more proper to display zen browser lab or mozilla lab

**Comment by adam-adrian** [comment]
No, because it's (Mozilla) Firefox thing originally, not what Zen did.

**Comment by Makitoid** [comment]
u mean u won't change it?

**Comment by adam-adrian** [comment]
It's up to the developers

**Comment by Makitoid** [comment]
> No, because it's (Mozilla) Firefox thing originally, not what Zen did.

i know that
